CHANGELOG.md

```diff
@@ -9,6 +9,8 @@
 
 ### Bug Fixes
 
+- [#1588](https://github.com/lapce/lapce/issues/1588): Fixed an issue where new windows would never be created after closing all windows on macOS.
```

Comment:
Sorry for the previous mistake, it's more appropriate to put the link to the pull request here. 😢

Reply:
Is this serious? Should I fix it with a new PR?